Source/WebKit/UIProcess/ios/WKContentViewInteraction.mm

```diff
@@ -1763,6 +1763,18 @@ - (BOOL)_touchEventsMustRequireGestureRecognizerToFail:(UIGestureRecognizer *)ge
         && [uiDelegate _webView:webView touchEventsMustRequireGestureRecognizerToFail:gestureRecognizer];
 }
 
+- (BOOL)_gestureRecognizerCanBePreventedByTouchEvents:(UIGestureRecognizer *)gestureRecognizer
+{
+    auto uiDelegate = static_cast<id<WKUIDelegatePrivate>>(self.webView.UIDelegate);
+    if (!uiDelegate)
+        return NO;
```

Comment:
This check is unnecessary. If `uiDelegate` is `nil`, the `respondsToSelector` check below will fail.

The method could just `return [uiDelegate respondsToSelector:@selector(_webView:gestureRecognizerCanBePreventedByTouchEvents:)] && [uiDelegate _webView:self.webView gestureRecognizerCanBePreventedByTouchEvents:gestureRecognizer]`.